contrib/endpoints/src/api_manager/context/service_context.cc

```diff
@@ -56,7 +60,11 @@ ServiceContext::ServiceContext(std::unique_ptr<ApiManagerEnvInterface> env,
       is_auth_force_disabled_(config_->server_config() &&
                               config_->server_config()
                                   ->api_authentication_config()
-                                  .force_disable()) {
+                                  .force_disable()),
+      is_check_security_rules_disabled_(config_->server_config() &&
```

Comment:
it is better to be disabled by default for now.  and only enable it when server_config has api_check_securty_rules_config and its firebase_server is specified. 

we can change it when service config is ready.